wordpress.yaml

```diff
@@ -1,7 +1,104 @@
 ---
 - name: Deploy Wordpress
   hosts: all
+  become: yes
   tasks:
-
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna be runing on the VM'
+    - name: Update & Upgrade the System
+      apt:
+        update_cache: yes
+        upgrade: yes    
+    
+    - name: Install LEMP Packages
+      apt:
+        name:
+            - nginx
+            - mysql-server
+            - php7.4-cli
+            - php7.4-fpm
+            - php7.4-mysql
+            - php7.4-json
+            - php7.4-opcache
+            - php7.4-mbstring
+            - php7.4-xml
+            - php7.4-gd
+            - php7.4-curl
+            
+    - name: Allow NGINX on Firewall   
+      ufw:
+        rule: allow
+        name: "Nginx Full"
+    
+    - name: Copy Nginx config       
+      copy:
+        src: /vagrant/wordpressCfg.txt
+        dest: /etc/nginx/sites-available
+        
+    - name: Symbolic link on Wordpress nginx configuration file      
+      file:
+        src: /etc/nginx/sites-available/wordpressCfg.txt
+        dest: /etc/nginx/sites-enabled/wordpressCfg.txt
+        state: link
+        
+    - name: Unlink Default nginx config      
+      file:
+        path: /etc/nginx/sites-enabled/default
+        state: absent
+        
+    - name: Create Wordpress Database   
+      shell: 'mysql < wpDB.sql'
+      args:
+        chdir: /vagrant/
+        
+    - name: Download Wordpress        
+      get_url:
+        url: https://wordpress.org/latest.tar.gz
+        dest: /var/www/
+        
+    - name: Extract Wordpress file        
+      unarchive:
+        src: /var/www/wordpress-5.8.2.tar.gz
+        dest: /var/www/
+    
+    - name: Copy wordpress config php file (wp database connection)        
+      copy:
+        src: /vagrant/wp-config.php
+        dest: /var/www/wordpress/
+        force: yes
+    
+    - name: Setup owner and group privileges to www-data and 755 permissions.       
+      file:
+        path: /var/www/wordpress
+        state: directory
+        recurse: yes
+        owner: www-data
+        group: www-data
+        mode: 0755
+            
+    - name: Download WP-Cli file      
+      get_url:
+        url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+        dest: /usr/local/bin/wp
+        mode: 0755
+    
+    - name: Install WP-CLI     
+      shell: 'wp core install --url="192.168.50.2" --title="Wordpress" --admin_user=admin --admin_email=admin@admin123.com --admin_password="!2three456." --path=/var/www/wordpress --allow-root'
```

Comment:
Can you please define variables for:
* URL
* Title
* Admin user
* Admin password
* Admin email
* Wordpress path
* DB user
* DB password
* DB name
* DB host

So that you would be easily read them from separate variables file

Reply:
What URL and Title?